Content.Server/Fax/FaxSystem.cs

```diff
@@ -51,6 +52,7 @@ public sealed class FaxSystem : EntitySystem
     [Dependency] private readonly MetaDataSystem _metaData = default!;
     [Dependency] private readonly FaxecuteSystem _faxecute = default!;
     [Dependency] private readonly EmagSystem _emag = default!;
+    [Dependency] private readonly IEntityManager entManager = default!;
```

Comment:
for private dependencies you need to use "_" in the name
```suggestion
    [Dependency] private readonly IEntityManager _entManager = default!;
```